examples/custom-clone.js

```diff
@@ -0,0 +1,37 @@
+export const g = {"Queue": null, "Link": null, "stringify": null, "Button": null,
+                  "createElement": null, "useRecordContext": null, "useCreatePath": null}
```

Comment:
We start by defining a global object that contains all the react functions etc. that we want as the keys, with values initialised to null.